treescript/src/treescript/git.py

```diff
@@ -42,6 +42,7 @@ async def checkout_repo(config, task, repo_path):
     if branch:
         # GitPython cannot simply `git checkout` to right upstream branch. We have to manually
         # create a new branch and manually set the upstream branch
+        log.info('Checking out branch "{}"'.format(branch))
```

Comment:
Nit: s/`/"/g to be consistent with the others, unless we specifically want the curly brackets for logging the branch.

Reply:
Black reformatted that line. I'll stick to it.